config/initializers/delayed_job_config.rb

```diff
@@ -10,6 +10,17 @@
 
 Delayed::Worker.delay_jobs          = !Rails.env.test?
 
+if ENV['HEROKU'].present?
+  Delayed::Worker.logger = Logger.new(STDOUT)
+else
+  Delayed::Worker.logger = Logger.new(File.join(Rails.root, 'log', 'delayed_job.log'))
+end
+
+if Delayed::Worker.delay_jobs && ($PROGRAM_NAME =~ /delayed_job/)
+  ActiveRecord::Base.logger = Delayed::Worker.logger
+  Rails.logger = Delayed::Worker.logger
```

Comment:
I dint get this. This will put all logs from `production.log` to `delayed_job.log` for anything run from `delayed_job` workers.
So that is two places where we would need to debug information. Is this expected?